e2e/pages/adf/material/sliderPage.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Util = require('../../../util/util');
+import { browser, protractor } from 'protractor';
+
+export class SliderPage {
```

Comment:
Is this class needed. Shouldn't the content of the class be part of sliderFilterPage.ts?

Reply:
Since we might use the slider somewhere else in the future, I think it's better to separate the methods related to the slider from the filter. What do you think ?

Reply:
It's better to doublecheck with Denys, but i think the code for search filters will not be used in other components. It's a separate code from form widgets code for example, so there might be differences related to locators or other things.

Reply:
All search widgets are unique and not shared across other components so far.